packages/test/end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -336,7 +333,6 @@ const tests = (args: ICompatTestArgs) => {
         const router = await dataStore.context.containerRuntime.createDataStore([testDataStoreType]);
         const comp2 = await requestFluidObject<ITestFluidObject>(router, "/");
 
-        // eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
What happened to this rule?

Reply:
The rule is still enabled, so it was either a false positive before or a false negative now. Not sure which.